### ISSUE TITLE
Unable to configure gradel if more than 1 flavor dimension

### ISSUE THREAD

**Issue by frnusmartobjects** [mannodermaus/android-junit5#128]
If you have an Android project which defines more than 1 flavor dimension Gradle fails the configuration phase. 
For example with:

```
flavorDimensions "firstDimension", "secondDimension"
productFlavors {
    firstFlavor {
        dimension "firstDimension"
    }
    secondFlavor {
        dimension "secondDimension"
    }
}
```

The following error is shown in console:

> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem occurred configuring project ':app'.
> > Extension with name 'firstFlavorSecondFlavorFilters' does not exist. Currently registered extension names: [ext, filters, debugFilters, releaseFilters, firstFlavorFilters, secondFlavorFilters, firstFlavorSecondFlavorDebugFilters, firstFlavorSecondFlavorReleaseFilters]
> 
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> CONFIGURE FAILED in 0s

When the plugin is removed then it configures the project correctly. It is somehow adding a "Filter" flavor to the process

**Comment by mannodermaus** [comment]
This is a previously fixed issue which I'm unable to reproduce using the latest version of the plugin. Please update your build script and it should start working. Feel free to reopen if it doesn't - if you could, a small sample application that exemplifies the issue would be very helpful.

**Comment by frnusmartobjects** [comment]
I'll try to configure a project that reproduces the problem. Could you point me to the aforementioned issue?

**Comment by mannodermaus** [comment]
Sure - the initial ticket for this was https://github.com/mannodermaus/android-junit5/issues/110, and its fix was part of the `1.3.1.0` release.
